LFGBulletinBoard/RequestList.lua

```diff
@@ -824,12 +834,14 @@ function GBB.ParseMessage(msg,name,guid,channel)
 
 end
 function GBB.UnfoldAllDungeon()
-	wipe(GBB.FoldedDungeons)
+	for k,v in pairs(GBB.FoldedDungeons) do
+		GBB.FoldedDungeons[k]=false
+	end
```

Comment:
This also has the side effect of assigning GBB.FoldedDungeons a new pointer every time fold all is toggled. That doesn't currently matter, but it could at some point. 

Since it iterates over every dungeon in UpdateList I think this change would optimize it from O(2n) to O(n). I don't think that's worth it given that UpdateList is already called frequently as is.

I'll hold off on this one, but I did test the changes anyway and they do work if you decide to add them.

Reply:
O(2N) is O(N) but really in this case it's O(1). But yea I don't think it's worth the tiny optimization

Reply:
The reset goes from O(n) to O(1) but the update stays O(n). So the function goes from O(n+n) to O(n+1) which, as you said, are the same.

Reply:
I mean that there is no real varying size here at least when it comes to dungeons, it's fairly constant. Number of requests does vary but ultimately it's also a small number where there is no hardly any difference between O(log(N)), O(N), or O(N^2) but you are right